### DISCUSSION TITLE
HTML tags and Links in Annotations

### DISCUSSION THREAD

**Discussion by Zapallar** [annotorious/annotorious#425] [General]
Hi, is there any way to add simple HTML tags or links (or links of any kind) to annotations?  Right now, when I'm adding them like this:
`"value": "Annotation text<br><br><a href='https...`
The HTML code is shown as plain text instead of a clickable Link ... what am I doing wrong?

**Comment by rsimon** [comment]
Hi, I'm afraid the comment popup escapse all HTML markup. (I believe that links in the plaintext, i.e. anything that starts with http[s]://, are made clickable. But would have to double-check. Could be that I only added this to a customization, but not the default editor...)

**Reply by Zapallar** [reply]
Tausend Dank Rainer. So there's also no chance to add line breaks like` <br> `etc? Is there any workaround to that issue incl. the clickable link? :)

**Reply by Zapallar** [reply]
Thing is, on a large zoomify map I'm annotating I would need certain annotations to have bold text, some other need links to other websites, etc ...

**Reply by rsimon** [reply]
Understood. But, sorry, there's no rich text popup available. I may eventually add one as an extension to the next major release (v3). But that depends on when I get around to doing it and/or if there's a client who needs it. For now you'd either have to live with the plaintext popup, or modify the popup yourself.

(FWIW: line breaks do work in the current popup. It supports plaintext only, but uses `<pre>` formatting, which means line breaks should be retained.)
